lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -125,10 +125,26 @@ def add_amount(post, amount)
         }
       end
 
+      def add_ach(post, payment)
+        post[:paymentInformation][:bank] = {
+          account: {
+            type: ach_account_type(payment.account_type),
+            number: payment.account_number
+          },
+          routingNumber: payment.routing_number
+        }
+      end
+
+      def ach_account_type(account_type)
+        account_type == 'checking' ? 'C' : 'S'
+      end
+
```

Comment:
🛑 Warning:

Is important to scrub the `account_number` and `routing_number` on the request.

Reply:
I checked that and the answer from the gateway is not returning attributes like `account_number` and `routing_number`,  or any sensitive attribute that the current scrubbing process was not handling.

Reply:
let's add the number itself to the scrub assertion anyway just to verify current scrubbing handles these values

Reply:
@edgarv09 scrubbing works for both, requests and responses ... in fact, Is more important for requests because inside the request body is where the CC numbers, CVVs, Cryptograms, and Bank Accounts are.

Reply:
Nice, thanks for the explanation. 
FYI
It's done the scrubbing test.